### ISSUE TITLE
Feature: Post-It sections/frames, which are own Excalidraws again

### ISSUE THREAD

**Issue by Kongolan** [excalidraw/excalidraw#5900]
There is one feature on miro.com (which is a good tool for visual graphs/presentation/process planning), which is pretty helpful. You can have a big map, with many subsections, which are contained in a frame like a picture or selectable on their own. I would call it post-it like frames, because they feel like you just work on many different things on one big visual board, where you zoom in the part, which interests you most.

I think this would be super helpful, if Excalidraw has a similar workflow. Even if you just add a post-it background or frame, which we can freely draw. So it's mainly a visual expansion. But if you try it out and might feel, you should make them full subsections, which are self-contained, it might would be a great idea.

See it as brain strom suggestion. I don't like to use miro.com if Excalidraw can do the same! 👍 Thanks for making the best plugin in Obsidian!

**Comment by 7flash** [comment]
Agree. Regarding performance, my approach is to show screenshot of the nested document which is being replaced with the actual elements when zooming close enough.
